doc/examples/fiber_to_bundle_coherence.py

```diff
@@ -0,0 +1,281 @@
+rotationally invariant processing, the discrete orientations 
+are required to be equally distributed over a sphere. By default, a sphere with 
+100 directions is used obtained from electrostatic repulsion in DIPY.
+"""
+
+# compute lookup table
+from dipy.denoise.enhancement_kernel import EnhancementKernel
+D33 = 1.0
+D44 = 0.02
+t = 1
+k = EnhancementKernel(D33, D44, t)
+
+"""
+The FBC measures are now computed, taking the tractography results and the lookup
+tables as input. 
+"""
+
+# apply FBC measures
+from dipy.tracking.fbcmeasures import FBCMeasures
+from time import time
+a = time()
+fbc = FBCMeasures(streamlines, k)
+
+"""
+After calculating the FBC measures, a threshold can be chosen on the relative
+FBC (RFBC) in order to remove spurious fibers. Recall that the relative FBC (RFBC)
+is calculated by the minimum of the moving average LFBC along the fiber. In this 
+example we show the results for threshold 0 (i.e. all fibers are included) and 
+0.2 (removing the 20 percent most spurious fibers).
+"""
+
+# calculate LFBC for original fibers
+fbc_sl_orig, clrs_orig, rfbc_orig = \
+  fbc.get_points_rfbc_thresholded(0, emphasis=0.01)
+
+# apply a threshold on the RFBC to remove spurious fibers
+fbc_sl_thres, clrs_thres, rfbc_thres = \
+  fbc.get_points_rfbc_thresholded(0.125, emphasis=0.01)
+
+"""
+The results of FBC measures are visualized, showing the original fibers colored
+by LFBC, and the fibers after the cleaning procedure via RFBC thresholding.
+"""
+
+# visualize the results
+from dipy.viz import fvtk
+from dipy.viz.colormap import line_colors
+from dipy.viz import window, actor
+
+ren = fvtk.ren()
+
+# original lines colored by LFBC
+lineactor = actor.line(fbc_sl_orig, clrs_orig, linewidth=0.2)
+fvtk.add(ren, lineactor)
+
+# horizontal (axial) slice of T1 data
+vol_actor1 = fvtk.slicer(t1_data, affine=affine)
+vol_actor1.display(None, None, 20)
+fvtk.add(ren, vol_actor1)
+
+# vertical (sagittal) slice of T1 data
+vol_actor2 = fvtk.slicer(t1_data, affine=affine)
+vol_actor2.display(35, None, None)
+fvtk.add(ren, vol_actor2)
+
+# show original fibers
+fvtk.camera(ren, pos=(-264, 285, 155), focal=(0, -14, 9), viewup=(0, 0, 1), 
+            verbose=False)
+fvtk.record(ren, n_frames=1, out_path='OR_before.png', size=(900, 900))
+
+# show thresholded fibers
+fvtk.rm(ren, lineactor)
+fvtk.add(ren, actor.line(fbc_sl_thres, clrs_thres, linewidth=0.2))
+fvtk.record(ren, n_frames=1, out_path='OR_after.png', size=(900, 900))
+
+"""
+
+.. figure:: OR_before.png
+   :align: center
+
+   The optic radiation obtained through probabilistic tractography colored by 
+   local fiber to bundle coherence.
+
+.. figure:: OR_after.png
+   :align: center
+
+   The tractography result is cleaned (shown in bottom) by removing fibers with 
+   a relative FBC (RFBC) lower than the threshold tau=0.2.
+
+Acknowledgements
+~~~~~~~~~~~~~~~~~~~~~~
+Funded by the European Research Council under the European Community's Seventh 
```

Comment:
I have removed the funding acknowledgement here. If we make a future paper about this we can mention it there I guess, but it's already in the abstracts.